packages/realm-react/src/__tests__/mocks.ts

```diff
@@ -19,6 +19,7 @@
 import { act } from "@testing-library/react-native";
 import { EstimateProgressNotificationCallback, ProgressRealmPromise, Realm } from "realm";
 import { sleep } from "../helpers";
+import { SyncSession } from "../../../realm/dist/public-types/internal";
```

Comment:
This seems wrong 🤔

Reply:
Created PR for these: https://github.com/realm/realm-js/pull/6833